reconcile/utils/terrascript/cloudflare_client.py

```diff
@@ -21,6 +21,7 @@
     provider,
 )
 
+from reconcile.cli import TERRAFORM_VERSION
```

Comment:
why is `TERRAFORM_VERSION` a list? If there are multiple version, should we always use the first `TERRAFORM_VERSION[0]`?

Reply:
this did not start out as a list: #1361

Reply:
and changed to a list here: #4126

Reply:
ah so it is not really used as a list for the TF case but was made a list because of OC